ICAFIX/ReApplyFixPipeline.sh

```diff
@@ -663,7 +686,11 @@ main()
 			$FSLDIR/bin/imrm ${StudyFolder}/${Subject}/MNINonLinear/Results/${fMRIName}/${fMRIName}${hpStr}
 		fi
 		$FSLDIR/bin/imrm ${StudyFolder}/${Subject}/MNINonLinear/Results/${fMRIName}/${fMRIName}${hpStr}.ica/filtered_func_data
-        rm ${StudyFolder}/${Subject}/MNINonLinear/Results/${fMRIName}/${fMRIName}${hpStr}.ica/Atlas_hp_preclean.dtseries.nii
+		#TSC: should these be ${hpStr} or _hp${hp}?
```

Comment:
The key distinction is that this script supports the use of hp<0 (no filtering), in which case `${hpStr}` gets set earlier to the empty string.